tests/test_controls.py

```diff
@@ -173,6 +173,12 @@ def test_default_sensor_contorls(action, expected):
         look_down=habitat_sim.ActionSpec(
             "look_down", habitat_sim.ActuationSpec(amount=10.0)
         ),
+        rotate_sensor_clockwise=habitat_sim.ActionSpec(
```

Comment:
The action names and the expected result of executing the action also need to be added to the `default_sensor_control_testdata` list above for them to actually be tested.